setup.cfg

```diff
@@ -122,6 +122,7 @@ dev =
     pydantic[dotenv]
     check-manifest>=0.42
     rich
+    napari-error-monitor
```

Comment:
I'm not sure if we want to install this by default? I think at least it should be a big notice to the community across most channels before we do. Even if most people would opt in anyway, I think it'll be nice for them to know it's coming in the next release/once we merge this PR.

Reply:
>I'm not sure if we want to install this by default?

you might have missed that this is in the `dev` section *not* the install requires.  so this only applies for (mostly) core developers who clone, and then specifically run `pip install -e .[dev]`

> The formatting of the example bug report is a bit wonky but I don't know how much that's in your control @tlambert03.

it's an object meant for serialization and transmission over the internet, not so much human consumption.  given that it's a one-time thing, I'm not sure the time it would take to "beautify" that bundle is worth it.

> I think it'll be nice for them to know it's coming in the next release/once we merge this PR.

don't forget, nothing is "coming" for anyone unless they **manually** go seek out the napari-error-monitor package and install it.  That doesn't ship with napari, so there are _two_ steps of opt in.  1) decide you care about this and go seek out that plugin, 2) relaunch napari, and specifically click the yes button.  otherwise nothing happens at all.

Reply:
> you might have missed that this is in the dev section

Oops you're right I did miss that it was in `dev`. That's totally fine then - as long as it doesn't ship with `napari`. Still would be nice to get some messaging out there that it's available for use.

> given that it's a one-time thing, I'm not sure the time it would take to "beautify" that bundle is worth it.

Agreed.

Reply:
> Still would be nice to get some messaging out there that it's available for use.

yep, once it's in I'd like developers to use it a bit.  start getting some bug reports, then once we have a feel for it, and how we can use it, start announcing on image.sc for those who want to help napari out